### ISSUE TITLE
Create input konba component

### ISSUE THREAD

**Issue by brauliodiez** [Lemoncode/quickmock#18]
**Pending on trasnformer implementation to be merged**

Crea a input konba component

<img width="192" alt="image" src="https://github.com/Lemoncode/quickmock/assets/1457912/f862bd14-dfc2-4dab-86d8-2f56226f775e">

We have got the following input SVG component:

https://github.com/brauliodiez/konba-playground/blob/main/public/widgets/input.svg

Implement a React.konba component that takes into consideration resizing, take a look to the combobox konba implementation on Quick Mock

https://github.com/Lemoncode/quickmock/blob/main/src/common/components/front-components/combobox-shape.tsx

Add a solid white background to the input inner content.

Right now we will support dynamic resizing, but we won't bother about adding a text property.

A possible starting point:

```tsx
import React, { forwardRef } from 'react';
import { Group, Rect, Text } from 'react-konva';
import { ShapeConfig } from 'konva/lib/Shape';

interface InputShapeProps extends ShapeConfig {
  id: string;
  x: number;
  y: number;
  width: number;
  height: number;
  onSelected: (id: string) => void;
}

export const InputShape = forwardRef<any, InputShapeProps>(
  ({ x, y, width, height, id, onSelected, ...shapeProps }, ref) => {
    return (
      <Group
        x={x}
        y={y}
        onClick={() => onSelected(id)}
        ref={ref}
        {...shapeProps}
      >
        <Rect
          x={10}
          y={10}
          width={180}
          height={30}
          cornerRadius={5}
          stroke="black"
          strokeWidth={2}
          fill="none"
        />
        <Text
          x={20}
          y={30}
          text="Input"
          fontFamily="Comic Sans MS, cursive"
          fontSize={15}
          fill="gray"
        />
      </Group>
    );
  }
);
```

About styling we will create a separate task to extract common styles for all components (e.g. font used)
